src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -543,7 +550,17 @@ private async Task CheckJobCompletionStatus()
                 // all queries marked as complete, reindex job is done, check success or failure
                 if (_reindexJobRecord.QueryList.Keys.All(q => q.Status == OperationStatus.Completed))
                 {
-                    await UpdateParametersAndCompleteJob();
+                    // Perform a final check to make sure there are no resources left to reindex
```

Comment:
>Perform a final check to make sure there are no resources left to reindex [](http://example.com/codeflow?start=23&length=73)

Imagine frustration of the user after 3 days of reindexing your job failed with:
"1 resources failed to be reindexed" 

Do we communicate clearly what rest of resources are actually reindexed? 

What would happen if user submit same job, would it be 3 days job agian?

Is there are way for user to find out what resources giving trouble?

Should user have right to pick threshold for this?

Reply:
Removed the retry logic and updated the information returned to the user in f1bce27.